csv2rdf/thesession/AccountForReconciliation.ttl

```diff
@@ -0,0 +1,48 @@
+        "http://www.wikidata.org/entity/Q170412",
+        "http://www.wikidata.org/entity/Q170412"
+    ],
+    "tune_id": "https://thesession.org/tunes",
+    "alias": "http://www.wikidata.org/prop/direct/P742", #It's "pseudonym" that is "alias used by someone". But I am not sure can this be used following an tune which is not a person.
+    "name": "http://www.wikidata.org/prop/direct/P2561", #name
+    "event": "http://www.wikidata.org/prop/direct/P2561", #It's just the name of an event.
+    "dtstart": "http://www.wikidata.org/prop/direct/P580", 
+    "dtend": "http://www.wikidata.org/prop/direct/P582",
+    "venue": "http://www.wikidata.org/prop/direct/P276", #location
+    "address": "http://www.wikidata.org/prop/direct/P6375", #street address
+    "town": "http://www.wikidata.org/prop/direct/P131", #located in the administrative territorial entity
+    "area": "http://www.wikidata.org/prop/direct/P276", #location
+    "area_wiki": "http://www.wikidata.org/prop/direct/P276",
+    "country": "http://www.wikidata.org/prop/direct/P17", #country
+    "country_wiki": "http://www.wikidata.org/prop/direct/P17",
+    "coordinate": "http://www.wikidata.org/prop/direct/P625", #coordinate location
+    "recording_id": "https://thesession.org/recordings", #Not sure if this is appropriate. Because "https://thesession.org/" pattern is more of describing a class instead of a property.
+    "artist": "https://musicbrainz.org/doc/Recording#Artist", #Ich once recommended to use "performer P175" of Wikidata. But we cannot guarantee all the property values are performers, because what if there are some composers who don't perform... So we decided to reuse metadata from musicbrainz. Using <https://musicbrainz.org/doc/Recording#Artist> is more rigorous. 
+    "artist_url": "https://musicbrainz.org/doc/Recording#Artist",
+    "recording": "http://www.wikidata.org/prop/direct/P2561", #name of the recording
+    "recording_wiki": "http://www.wikidata.org/prop/direct/P2888", #exact match
+    "track": "http://www.wikidata.org/prop/direct/P658", #tracklist
+    "number": "http://www.wikidata.org/prop/direct/P1114", #quantity
+    "tune": "http://www.wikidata.org/prop/direct/P1625", #has melody: this work has the melody of the following work
+    "sessions_id": "https://thesession.org/sessions", # Warning: We use the <https://thesession.org/sessions> as both a class and a property, though it's not invalid.
+    "town_wiki": "http://www.wikidata.org/prop/direct/P131", #located in the administrative territorial entity
+    "date": "http://www.wikidata.org/prop/direct/P585", #point in time. It's appropriate becasue the property values are like: 3/6/2013 9:02:44 PM, 6/7/2016 3:57:16 PM...
+    "tuneset_id": "https://thesession.org/tuneset_id",
+    "username": "http://www.wikidata.org/prop/direct/P554", #website username or ID
+    "setting_id": "https://thesession.org/setting_id",
```

Comment:
I think this is problematic. 

Here's an example from the turtle output:
```
<https://thesession.org/members/1/sets/10129> a <http://www.wikidata.org/entity/Q36161> ;
    ns1:P136 "jig" ;
    ns1:P2308 <https://www.wikidata.org/wiki/Q1079270> ;
    ns1:P2561 "Connaughtman's Rambles, The",
        "Mouse In The Mug, The",
        "Sliabh Russell" ;
    ns1:P3440 <https://www.wikidata.org/wiki/Q50353378>,
        "6/8" ;
    ns1:P554 "Jeremy" ;
    ns1:P826 <https://www.wikidata.org/wiki/Q1124006>,
        <https://www.wikidata.org/wiki/Q5728362>,
        "A dorian",
        "D major",
        "G major" ;
    ns2:setting_id "19",
        "25104",
        "335" ;
    ns2:tunes <https://thesession.org/tunes/19>,
        <https://thesession.org/tunes/335>,
        <https://thesession.org/tunes/362> .
```

The subject is the tuneset "https://thesession.org/members/1/sets/10129". This set has three tunes in it: 

- <https://thesession.org/tunes/19> in setting <https://thesession.org/tunes/19#setting19>
- <https://thesession.org/tunes/335> in setting <https://thesession.org/tunes/335#setting335>
- <https://thesession.org/tunes/362> in setting <https://thesession.org/tunes/362#setting25104>

The settings that are part of this set have URI's on thesession.org that are not making their way to the turtle output.

Reply:
Right. They are not making their way to the turtle output because the original reconciled CSV file is like this:
![image](https://github.com/user-attachments/assets/f1f02257-ccd7-4f6d-82db-c308beae9972)

Please see the setting_id column. The property value is just literal. We haven't reconciled with this column because we don't know what the integer of setting_id's value stands for.

Reply:
This may involve a finer granularity of the data structure of The Session. I don't think we have to make it very clear right now. Thanks.

Reply:
> we don't know what the integer of setting_id's value stands for

I don't think this is true. They are an ID. A setting is an instance of a tune. A tune has a uri: `https://thesession.org/tunes/362`. A setting of that tune has a URI: `https://thesession.org/tunes/362#setting25104`. 

I'm just not sure why we would construct this data property with a literal value when we have an ID that corresponds to an object in the Session DB.